source/sass/style.scss

```
@@ -1,3 +1,34 @@
-body {
-  visibility: visible;
-}
+@import "./global/fonts.scss";
```

```suggestion
@import "./global/fonts";
```

расширения не нужны, особенность саса, у лесс, наоборот, надо расширения файлов.
Когда удалишь расширения (через заменить делай, .scss заменить на пустоту), заново отсортируй по-алфавиту. Тут легко видеть как надо сортировать, тк у тебя файлы в проекте тоже отсортированы по-алфавиту

---

классная фишка
я до этого через альт выделял строки и удалял